content/uk/docs/reference/glossary/container-runtime.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Container Runtime
+id: container-runtime
+date: 2019-06-05
+full_link: /docs/setup/production-environment/container-runtimes
+<!--The container runtime is the software that is responsible for running containers.-->
+short_description: >
```

Comment:
This line is causing [Netlify build to fail](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/630749b6b6303b00081e5a0e#L77).
> Error building site: "/opt/build/repo/content/uk/docs/reference/glossary/container-runtime.md:6:1": failed to unmarshal YAML: yaml: line 6: could not find expected ':'

```suggestion
full_link: /docs/setup/production-environment/container-runtimes
short_description: >
```